packages/jest-mock/src/index.ts

```diff
@@ -98,20 +100,6 @@ export type MaybeMockedDeep<T> = T extends FunctionLike
   ? MockedObjectDeep<T>
   : T;
 
-export type Mocked<T> = {
-  [P in keyof T]: T[P] extends FunctionLike
-    ? MockInstance<T[P]>
-    : T[P] extends ClassLike
-    ? MockedClass<T[P]>
-    : T[P];
-} & T;
-
-export type MockedClass<T extends ClassLike> = MockInstance<
```

Comment:
@SimenB Breaking here. I could split it to separate PR quickly for Jest 28 release, perhaps?

Reply:
Wait. Nope, this will no break. False alarm (;